packages/typography/src/Typography.tsx

```diff
@@ -112,22 +115,75 @@ function Body({ children, className, weight = 'regular', ...rest }: BodyProps) {
 
 Body.displayName = 'Body';
 
+const anchorDataProp = createDataProp('anchor-inline-code');
+
 const code = css`
+  background-color: ${uiColors.gray.light3};
+  border: 1px solid ${uiColors.gray.light1};
+  border-radius: 3px;
   font-family: ${fontFamilies.code};
-  display: inline-block;
+
+  ${anchorDataProp.selector}:hover > code > & {
+    border-color: ${uiColors.blue.base};
+  }
+`;
+
+const codeFocus = css`
+  ${anchorDataProp.selector}:focus > code > & {
```

Comment:
curious why these styles apply to the children rather than the whole element

Reply:
Because the border is implemented on the inner span so that it remains the same whether or not the code is wrapped in an anchor tag